products/redhat.md

```diff
@@ -11,32 +11,69 @@ releasePolicyLink: https://access.redhat.com/support/policy/updates/errata
 activeSupportColumn: true
 releaseDateColumn: true
 releaseColumn: false
-sortReleasesBy: 'release'
+sortReleasesBy: 'cycleShortHand'
 releaseLabel: "RHEL __RELEASE_CYCLE__"
+LTSLabel: "<abbr title='Extended Life Cycle Support'>ELS</abbr>"
 releases:
   - releaseCycle: "9"
     release: 2022-05-18
     support: 2027-05-31
     eol: 2032-05-31
     latest: "9.0"
+    cycleShortHand: 10
+  - releaseCycle: "9"
```

Comment:
looks like there is a typo here , shouldnt this has to be releaseCycle: "10"
because right now there are releaseCycle: "9" twice in list

Reply:
With commit https://github.com/endoflife-date/endoflife.date/pull/1254/commits/f99f6d59683410bbdc2c27f95afa9c127e6e6f11 it is distinguishing between ELS and non-ELS with the LTS labels, one of those is non-LTS (so not ELS), and the other one is. Although I can see how this can be confusing for contributors, so I'll add a comment for such releases.